misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -746,6 +745,36 @@ abstract class AnyAssertionsSpec(
         }
     }
 
+    prefixedDescribe("because") {
+        it("the test on the supplied subject is not throwing an assertion error") {
+            expect("filename")
+                .because("? is not allowed in file names on Windows") {
+                    containsNot("?")
+                }
+        }
+
+        it("provoke the failing of one assertion") {
+            expect {
+                expect("filename?")
+                    .because("? is not allowed in file names on Windows") {
+                        containsNot("?")
+                        startsWith("f")
+                    }
+            }.toThrow<AssertionError>()
```

Comment:
This test case is currently useless: It only tests that `containsNot` is working correctly. To make it meaningful, we should test that the thrown `AssertionError` contains the explanation, formatted exactly like we want it to be formatted.